x/datadeal/genesis.go

```diff
@@ -74,6 +80,18 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	}
 	genesis.DataDeliveryVotes = dataDeliveryVotes
 
-	// TODO: Add an ExportGenesis of DataVerification/Delivery Queue
+	dataVerificationQueues, err := k.GetAllDataVerificationQueue(ctx)
```

Comment:
How about change like this? We have only one DataVerificationQueue.
```suggestion
	dataVerificationQueue, err := k.GetDataVerificationQueue(ctx)
```